drivers/include/epd_bw_spi.h

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (C) 2019 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_epd_bw_spi Generic black/white e-paper/e-ink SPI display driver.
+ * @ingroup     drivers_actuators
+ * @brief       Device driver for black/white e-ink/e-paper SPI displays.
+ *
+ * TODO
+ *
+ * @{
+ * @file
+ * @brief       Generic black/white e-paper/e-ink display SPI driver.
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ */
+
+#ifndef EPD_BW_SPI_H
+#define EPD_BW_SPI_H
+
+#include "periph/spi.h"
+#include "periph/gpio.h"
+
+#ifdef MODULE_DISP_DEV
+#include "disp_dev.h"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define EPD_BW_SPI_COLOR_WHITE (0xFF)   /**< White (8x1 pixels) */
+#define EPD_BW_SPI_COLOR_BLACK (0x00)   /**< Black (8x1 pixels) */
+
+/**
+ * @brief   Data entry mode settings.
+ *
+ * This setting affect the automatic increment/decrement of the address counters.
+ */
+typedef enum {
+    EPD_BW_SPI_Y_DEC_X_DEC  = 0b00, /**< Y decrement, X decrement */
+    EPD_BW_SPI_Y_DEC_X_INC  = 0b01, /**< Y decrement, X increment */
+    EPD_BW_SPI_Y_INC_X_DEC  = 0b10, /**< Y increment, X decrement */
+    EPD_BW_SPI_Y_INC_X_INC  = 0b11, /**< Y increment, X increment */
```

Comment:
Standard C unfortunately doesn't define binary constants. Could you change these to use hex values instead?